test/test_performance.py.in

```diff
@@ -121,6 +123,10 @@ def generate_test_description(ready_fn):
         '-l', performance_log_prefix,
     ]
 
+    sync_env = {}
+    if '@SYNC_MODE@' == 'sync' and '@COMM@' != 'ROS2':
```

Comment:
> `'@COMM@' != 'ROS2'`

Is this a limitation of the `performance_test` package?

Reply:
Yes, This is a limitation https://github.com/ros2/performance_test/blob/master/performance_test/src/experiment_configuration/experiment_configuration.cpp#L166